src/modules/twap/containers/TwapFormWidget/tooltips.tsx

```diff
@@ -0,0 +1,56 @@
+export interface LabelTooltipObject {
+  label: string
+  tooltip?: JSX.Element
+}
+
+export interface LabelTooltipItems {
+  [key: string]: LabelTooltipObject
+}
+
+export const LABELS_TOOLTIPS: LabelTooltipItems = {
+  numberOfParts: {
+    label: 'No. of parts',
+    tooltip: <>Your TWAP order will be split into this many parts, which will execute one by one.</>,
+  },
+  totalDuration: {
+    label: 'Total duration',
+    tooltip: (
+      <>
+        The "Total duration" is the duration it takes to execute all parts of your TWAP order. For instance, your order
+        consists of 2 parts placed every 30 minutes, the total time to complete the order is 1 hour. Each limit order
+        remains open for 30 minutes until the next part becomes active.
```

Comment:
will we want to give some style to the values? like the `2 parts` or `30 minutes`

Reply:
Fixed